settings.gradle.kts

```
@@ -1,15 +1,15 @@
 plugins {
-    id("com.gradle.enterprise") version "3.17"
+    id("com.gradle.develocity") version "3.17"
     id("org.gradle.toolchains.foojay-resolver-convention") version "0.8.0"
 }
 
-val isCiBuild = System.getenv("CI") != null
-
-gradleEnterprise {
+develocity {
     buildScan {
-        termsOfServiceUrl = "https://gradle.com/terms-of-service"
-        termsOfServiceAgree = "yes"
-        publishAlwaysIf(isCiBuild)
+        termsOfUseUrl = "https://gradle.com/terms-of-service"
```

hmm, did the url change?

https://docs.gradle.com/enterprise/gradle-plugin/current/#connecting_to_scans_gradle_com

or is this something else?

---

`termsOfServiceUrl` has been replaced by `termsOfUseUrl`.

---

I meant the value of it, see the docs.

---

It looks like both are accepted for now.

---

Let's use the new one:
```suggestion
        termsOfUseUrl = "https://gradle.com/help/legal-terms-of-use"
```
It's in the develocity plugin docs + if you make a typo in it, it'll recommend this one.